bigquery_etl/shredder/config.py

```diff
@@ -272,6 +272,44 @@ def dataset_id(self):
     glean_target(table="org_mozilla_ios_lockbox_stable.baseline_v1"): IOS_LOCKBOX_SRC,
     glean_target(table="org_mozilla_ios_lockbox_stable.events_v1"): IOS_LOCKBOX_SRC,
     glean_target(table="org_mozilla_ios_lockbox_stable.metrics_v1"): IOS_LOCKBOX_SRC,
+    # activity stream
+    client_id_target(table="activity_stream_stable.events_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.cfr_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.onboarding_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_stable.snippets_v1"): DESKTOP_SRC,
+    client_id_target(table="activity_stream_stable.sessions_v1"): DESKTOP_SRC,
+    client_id_target(table="messaging_system_derived.cfr_users_daily_v1"): DESKTOP_SRC,
+    client_id_target(
+        table="messaging_system_derived.cfr_users_last_seen_v1"
+    ): DESKTOP_SRC,
+    client_id_target(
+        table="messaging_system_derived.onboarding_users_daily_v1"
+    ): DESKTOP_SRC,
+    client_id_target(
+        table="messaging_system_derived.onboarding_users_last_seen_v1"
+    ): DESKTOP_SRC,
+    client_id_target(
+        table="messaging_system_derived.snippets_users_daily_v1"
+    ): DESKTOP_SRC,
+    client_id_target(
+        table="messaging_system_derived.snippets_users_last_seen_v1"
+    ): DESKTOP_SRC,
+    impression_id_target(
+        table="activity_stream_stable.impression_stats_v1"
+    ): IMPRESSION_SRC,
+    impression_id_target(table="activity_stream_stable.spoc_fills_v1"): IMPRESSION_SRC,
+    impression_id_target(
+        table="messaging_system_stable.undesired_events_v1"
+    ): IMPRESSION_SRC,
+    impression_id_target(
+        table="messaging_system_derived.cfr_users_daily_v1"
+    ): IMPRESSION_SRC,
```

Comment:
```suggestion
    impression_id_target(
        table="messaging_system_stable.cfr_v1"
    ): IMPRESSION_SRC,
    impression_id_target(
        table="messaging_system_derived.cfr_users_daily_v1"
    ): IMPRESSION_SRC,
```

I think we also need to list cfr_v1 for both clientId and impression_id

Reply:
i need to update this so that a target is correctly handled with more than one source